README.rst

```diff
@@ -13,108 +13,160 @@
+- ...
 
-Dashboard
-   On overview of test result data will be displayed in the Dashboard
+There are also some charts which help you to get the the overview analysis about
+the test result, components's status or even the comparison to the previous test
+execution results.
 
-   |IMG_WEBAPP_DASHBOARD|
+.. image:: packagedoc/additional_docs/pictures/Dashboard.png
+   :alt: Dashboard view
 
 Data table
-   The data table allows you to
-      - browse into the single test results. 
-      - open the traceback for failed test cases.
-      - collaborate with your team.
-      - filter test cases.
+~~~~~~~~~~
+
+The data table show all executed test case and their results order by 
+component's name. Besides, it also allows to:
+
+- browse into the detail of single test results. 
+- open the traceback for failed test cases.
+- collaborate with your team by leaving your comment.
+- filter test cases.
+- search test cases.
    
-   |IMG_WEBAPP_DATATABLE|
+.. image:: packagedoc/additional_docs/pictures/DataTable.png
+   :alt: DataTable view
 
 Run-time view
-   The run-time view shows you test cases grouped by compnent. The size of a block is relative for the run-time of a test case. This allows you to optimize the run-time of a your test suite.
+~~~~~~~~~~~~~
 
-   |IMG_WEBAPP_RUNTIME|
-
-Diff view
-   The diff view allows you to diff test runs from different test suites.
-   For this a spiral view is used where the center is the start of the test suite execution.
-   
-   The diff view allows you to
-      - show only test cases which are other than passed.
-      - show only test cases which are failed.
-      - show only test cases which are new or missing.
-      - open the traceback for failed test cases.
-      - diff up to 5 versions
-      
-   |IMG_WEBAPP_DIFFVIEW|      
+The run-time view shows you test cases grouped by compnent. 
+The size of a block is relative for the run-time of a test case. 
+This allows you to optimize the run-time of a your test suite.
 
+.. image:: packagedoc/additional_docs/pictures/Runtime.png
+   :alt: Runtime view
 
-Installation
-------------
+Diff view
+~~~~~~~~~
 
+The diff view allows you to diff test runs from different test suites.
+For this a spiral view is used where the center is the start of the test suite 
+execution.
 
+The diff view allows you to:
 
-Example
--------
+- show only test cases which are other than passed.
+- show only test cases which are failed.
+- show only test cases which are new or missing.
+- open the traceback for failed test cases.
+- diff up to 5 versions.
+   
+.. image:: packagedoc/additional_docs/pictures/DiffView.png
+   :alt: Diff view    
 
+Contribution
+------------
+We are always searching support and you are cordially invited to help to improve 
+TRWebApp_ tool.
 
+Documentation
+-------------
+To understand more detail about the WebApp's features, the chart's meanings, 
+the displayed information on each view, please refer to 
+`TRWebApp’s Documentation`_.
```

Comment:
Hi Ngoan,
did you work on an update for this document, too?
I'm not sure, but I remember that you told this in our weekly sync.
Thank you,
 Thomas